contracts/ScheduledPaymentModule.sol

```diff
@@ -16,9 +19,17 @@ contract ScheduledPaymentModule is Module {
     );
     event PaymentScheduled(uint256 nonce, bytes32 spHash);
     event ScheduledPaymentCancelled(bytes32 spHash);
+    event ScheduledPaymentExecuted(bytes32 spHash);
     event ConfigSet(address config);
 
     error UnknownHash(bytes32 spHash);
+    error InvalidPeriod(bytes32 spHash);
+    error ExceedMaxGasPrice(bytes32 spHash);
```

Comment:
I suggest `MaxGasPriceExceed` to be consistent with other event names

Reply:
It's not an event. It's an error. Should I still rename it consistent with the events?

Reply:
@jurgenwerk should I update this? Or is that ok to have a different format between error and event?